src/DisplayCalendar.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prop-types */
```

Comment:
You have split it up into 3 components.

1. Week - Always comprises of 7 days. Visually it is a single row in the calendar.
2. Days - Can have a visible, innerText, position(may be) property.
3. DisplayCalender - It will be a manager that will render Weeks and Days Components with their respective props.